### PR TITLE
chore: docker-compose postgres healthcheck

### PR DESCRIPTION
**Pull request by patriknw**
* instead of sleep in CI

### CODE REVIEW

File: .github/workflows/integration-tests-r2dbc.yml [hunk 1]
```diff
@@ -48,9 +48,7 @@ jobs:
 
       - name: Start DB
         run: |-
-          docker-compose -f docker-files/docker-compose-postgres.yml up -d
-          # TODO: could we poll the port instead of sleep?
-          sleep 10
+          docker-compose -f docker-files/docker-compose-postgres.yml up --build --detach --wait
```
[the remark this answers is not in the document]
Comment (pvlugter) [reply .github/workflows/integration-tests-r2dbc.yml:51]: Yeah, `--build` is only if you need it to first build the image from a Dockerfile.

**Comment by patriknw** [comment]
Same problem as in https://github.com/akka/akka-persistence-r2dbc/pull/401

**Comment by pvlugter** [comment]
Have got this working for https://github.com/akka/akka-persistence-r2dbc/pull/437. Where `docker-compose` -> v1 and `docker compose` -> v2. Will update this PR directly.

**Review by pvlugter — APPROVED** [review]
Working now

File: akka-distributed-cluster-docs/src/main/paradox/guide/2-service-to-service.md [hunk 2]
```diff
@@ -248,7 +248,7 @@ your own workstation. Docker, a JDK and @java[maven]@scala[sbt] is all that need
 1. Start a local PostgresSQL server on default port 5432. The included `docker-compose.yml` starts everything required for running locally.
 
     ```shell
-    docker-compose up -d
+    docker-compose up --wait
```
Comment (patriknw) [inline akka-distributed-cluster-docs/src/main/paradox/guide/2-service-to-service.md:251]: these are still on the old `docker-compose`

**Review by patriknw — COMMENTED** [review]
still some old `docker-compose`, I can update those

File: akka-distributed-cluster-docs/src/main/paradox/guide/2-service-to-service.md (hunk 2 above)
Comment (patriknw) [reply akka-distributed-cluster-docs/src/main/paradox/guide/2-service-to-service.md:251]: ok, confusing

Comment (pvlugter) [reply akka-distributed-cluster-docs/src/main/paradox/guide/2-service-to-service.md:251]: Actually, maybe on linux distros generally there can be both `docker-compose` v1 and `docker compose` v2? So probably good to update to `docker compose` everywhere.

**Pull request merged**